src/xlsx/mod.rs

```diff
@@ -197,6 +197,29 @@ pub struct Xlsx<RS> {
     pictures: Option<Vec<(String, Vec<u8>)>>,
     /// Merged Regions: Name, Sheet, Merged Dimensions
     merged_regions: Option<Vec<(String, String, Dimensions)>>,
+    /// Reader options
+    options: XlsxOptions,
+}
+
+/// Xlsx reader options
+#[derive(Debug, Default)]
+pub struct XlsxOptions {
+    /// By default, calamine skips empty rows until a nonempty row is found
+    pub keep_first_empty_rows: bool,
```

Comment:
I think a more generic approach would be to have the option to set header row?
By default `None` means that it'll skip all empty rows (like today) and then we could have `with_header_row(self, row: u32)`?
I understand that >90% of use cases are the first row but it still feels better and not necessarily more complicated to give the option to the users?

Reply:
Perfect will do that

Reply:
For what it's worth, I prefer the option to load the empty rows before the header row. I am processing unknown/volatile files and I know what row the header should be, what the column names I am expecting, and I do my own processing to resolve duplicate column names. My desire to be able to load the file as-is, figure out where the headers are, possibly report that they were on a different row than expected, identify headers on the fly as needed, etc. Sometimes there are rows before the header that are empty, sometimes there are rows that are not empty.

Reply:
Done